wsd/FileServer.cpp

```diff
@@ -1414,6 +1414,7 @@ FileServerRequestHandler::ResourceAccessDetails FileServerRequestHandler::prepro
     Poco::replaceInPlace(preprocess, std::string("%ZOTERO_ENABLED%"), boolToString(config.getBool("zotero.enable", true)));
     Poco::replaceInPlace(preprocess, std::string("%DOCUMENT_SIGNING_ENABLED%"), boolToString(config.getBool("document_signing.enable", true)));
     Poco::replaceInPlace(preprocess, std::string("%WASM_ENABLED%"), boolToString(ConfigUtil::getConfigValue<bool>("wasm.enable", false)));
+    Poco::replaceInPlace(preprocess, std::string("%SLIDESHOW_ENABLED%"), boolToString(ConfigUtil::getConfigValue<bool>("slideshow", false)));
```

Comment:
1. last parameter should be true - we want to enable it by default when config option is missing - see my comment in the config file
2. `%SLIDESHOW_ENABLED%` -> `%CANVAS_SLIDESHOW_ENABLED%`